Library/Formula/td.rb

```diff
@@ -0,0 +1,49 @@
+require "language/go"
+
+class Td < Formula
+  homepage "https://github.com/Swatto/td"
+  revision 1
+
+  stable do
+    url "https://github.com/Swatto/td/archive/1.2.0.tar.gz"
+    sha256 "86a84cb183e2b3700a23cc5570b65ea8d133e51c5c59e321929ace727253d55b"
+  end
```

Comment:
This doesn't need to be in a `stable` block; these can just be at the class level.